src/components/app/Modal.js

```diff
@@ -180,10 +183,15 @@ class Modal extends Component {
     }
 }
 
+const mapStateToProps = state => ({
```

Comment:
Why did you connect Modal component to redux, if the parent already has that variable, and we can pass it as props?